html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title>HTML Skeletons</title>
+    </head>
+    <body>
+      Hello, HTML!
+    </body>
+  </html>
```

Comment:
The `html` element doesn't need to be indented, since it is not a child of any element. It can be at the same level of indentation at the `DOCTYPE` declaration

Reply:
![Screen Shot 2021-10-04 at 12 44 47 PM](https://user-images.githubusercontent.com/90732703/135916996-f8e33238-634c-471e-b708-b6cf76664a05.png)

Reply:
Changes were made.